runtime/autoload/health/provider.vim

```diff
@@ -497,6 +485,72 @@ function! s:check_for_pyenv() abort
   return [pyenv_path, pyenv_root]
 endfunction
 
+" Locate Python executable by running invocation and checking
+" sys.executable.
+function! s:locate_pythonx(invocation) abort
+  return s:normalize_path(system(a:invocation
+    \ . ' -c "import sys; sys.stdout.write(sys.executable)"'))
+endfunction
+
+" If $VIRTUAL_ENV is set, check whether its Python executables will be
+" the first on the $PATH of both Neovim and subshells spawned from
+" Neovim.
+function! s:check_active_virtualenv() abort
+  call health#report_start('Python active virtualenv')
+  let errors = []
+  " hints are kept as Dictionary keys in order to discard duplicates
+  let hints = {}
+  " the virtualenv should contain some Python executables, and those
+  " executables should be first both on Neovim's path and on the path of
+  " subshells launched from Neovim
+  let venv_pythonxs = glob($VIRTUAL_ENV . '/bin/python*', v:true, v:true)
```

Comment:
This is always false on Windows. python binaries are at `$VIRTUAL_ENV/Scripts/` if creating virtualenv via `venv` module. If this function cannot check on Windows, skip all virtualenv checks for it.

Reply:
Hmm, what about this? Or will it be slow-ish?

```diff
-let venv_pythonxs = glob($VIRTUAL_ENV . '/bin/python*', v:true, v:true)
+let venv_pythonxs = glob($VIRTUAL_ENV . '/*/python*', v:true, v:true)
```

Reply:
Oh snap, good catch! I don't really use Python on Windows except under WSL, so I always forget about this. Does d6025c0897232abff2adff9c786ba9117a3ea185 look reasonable?

If `has('win32')` is not entirely reliable for detecting whether Neovim is running under Windows, then @lithammer's suggestion looks good, or maybe `'/{Scripts,bin}/python*'` if this glob syntax is supported in Neovim, that way speed shouldn't be a concern.

Reply:
Just did a quick test and `'/{Scripts,bin}/python*'` seems like it should work.